open_duck_mini_v2/open_duck_mini_v2.xml

```diff
@@ -0,0 +1,454 @@
+            <body name="knee_and_ankle_assembly" pos="0.07415 0 0.03511" quat="0 0.707107 0 -0.707107">
+              <inertial pos="0.00253369 -0.0390636 0.0102776" quat="0.576793 0.413738 -0.51125 0.484515" mass="0.12407"
+                diaginertia="0.000228502 0.000216625 7.02746e-05"/>
+              <joint name="left_hip_pitch" pos="0 0 0" axis="0 0 1" range="-1.22173 0.523599"/>
+              <geom pos="0.01606 0.065 0.10915" quat="0.707107 -0.707107 0 0" type="mesh"
+                rgba="0.917647 0.917647 0.917647 1" mesh="left_cache"/>
+              <geom pos="0.01606 0.14365 0.10925" quat="0.707107 -0.707107 0 0" type="mesh"
+                rgba="0.647059 0.647059 0.647059 1" mesh="leg_spacer"/>
+              <geom pos="0.01606 0.14365 0.10915" quat="0.707107 -0.707107 0 0" type="mesh"
+                rgba="0.223529 0.219608 0.219608 1" mesh="left_knee_to_ankle_right_sheet"/>
+              <geom pos="0.01606 0.14365 0.10915" quat="0.707107 -0.707107 0 0" type="mesh"
+                rgba="0.223529 0.219608 0.219608 1" mesh="left_knee_to_ankle_left_sheet"/>
+              <geom pos="0 -0.05315 0.00955" quat="0.5 -0.5 -0.5 0.5" type="mesh" rgba="0.247059 0.243137 0.243137 1"
+                mesh="wj-wk00-0123middlecase_56"/>
+              <geom pos="0 -0.05315 0.00955" quat="0.5 -0.5 -0.5 0.5" type="mesh" rgba="0.623529 0.666667 0.701961 1"
+                mesh="drive_palonier"/>
+              <geom pos="0 -0.05315 0.00955" quat="0.5 -0.5 -0.5 0.5" type="mesh" rgba="0.247059 0.243137 0.243137 1"
+                mesh="wj-wk00-0124bottomcase_45"/>
+              <geom pos="0 -0.05315 0.00955" quat="0.5 -0.5 -0.5 0.5" type="mesh" rgba="0.247059 0.243137 0.243137 1"
+                mesh="wj-wk00-0122topcabinetcase_95"/>
+              <geom pos="0 -0.05315 0.00665" quat="0.5 -0.5 -0.5 0.5" type="mesh" rgba="0.623529 0.666667 0.701961 1"
+                mesh="passive_palonier"/>
+              <body name="knee_and_ankle_assembly_2" pos="0 -0.07865 0.0001">
+                <inertial pos="5.01859e-06 -0.0577465 0.0181136" quat="0.708422 0.705789 1.07934e-05 0.00022447"
+                  mass="0.07259" diaginertia="4.99575e-05 4.23935e-05 1.8727e-05"/>
+                <joint name="left_knee" pos="0 0 0" axis="0 0 1" range="-1.5708 1.5708"/>
+                <geom pos="0.01606 0.14365 0.10925" quat="0.707107 -0.707107 0 0" type="mesh"
+                  rgba="0.647059 0.647059 0.647059 1" mesh="leg_spacer"/>
+                <geom pos="0.01606 0.14365 0.10915" quat="0.707107 -0.707107 0 0" type="mesh"
+                  rgba="0.223529 0.219608 0.219608 1" mesh="left_knee_to_ankle_right_sheet"/>
+                <geom pos="0.01606 0.14365 0.10915" quat="0.707107 -0.707107 0 0" type="mesh"
+                  rgba="0.223529 0.219608 0.219608 1" mesh="left_knee_to_ankle_left_sheet"/>
+                <geom pos="0 -0.05315 0.00955" quat="0.5 -0.5 -0.5 0.5" type="mesh" rgba="0.247059 0.243137 0.243137 1"
```

Comment:
I'd prefer if `rgba`s were converted to materials and material names used instead.